description: a virtual destructor is called without this patch
origin: https://chromium-review.googlesource.com/c/chromium/src/+/1645366

--- a/chrome/browser/performance_manager/render_process_user_data.cc
+++ b/chrome/browser/performance_manager/render_process_user_data.cc
@@ -123,4 +123,9 @@ void RenderProcessUserData::RenderProces
   host->RemoveUserData(kRenderProcessUserDataKey);
 }
 
+void RenderProcessUserData::RenderProcessHostDestroyed(
+    content::RenderProcessHost* host) {
+  host->RemoveUserData(kRenderProcessUserDataKey);
+}
+
 }  // namespace performance_manager
